docs/fundamentals/orleans.md

```diff
@@ -0,0 +1,192 @@
+
+// Add the resources which we will use for Orleans clustering and grain state storage.
+var storage = builder.AddAzureStorage("storage").RunAsEmulator();
+var clusteringTable = storage.AddTables("clustering");
+var grainStorage = storage.AddBlobs("grainstate");
+
+// Add the Orleans resource to the Aspire DistributedApplication builder, then configure it with Azure Table Storage for clustering
+// and Azure Blob Storage for grain storage.
+var orleans = builder.AddOrleans("my-app")
+                     .WithClustering(clusteringTable)
+                     .WithGrainStorage("Default", grainStorage);
+
+// Reference the Orleans resource from the 'silo' project so that it can join the Orleans cluster as a service.
+builder.AddProject<Projects.OrleansServer>("silo")
+       .WithReference(orleans)
+       .WithReplicas(3);
+
+// Reference the Orleans resource as a client from the 'frontend' project so that it can connect to the Orleans cluster.
+builder.AddProject<Projects.OrleansClient>("frontend")
+       .WithReference(orleans.AsClient())
+       .WithExternalHttpEndpoints()
+       .WithReplicas(3);
+
+// Build and run the application.
+using var app = builder.Build();
+await app.RunAsync();
+```
+
+To consume the Aspire Orleans resource from an Orleans server project, there are a few steps:
+
+1. Add the relevant Aspire components. In our example, we are using _Aspire.Azure.Data.Tables_ and _Aspire.Azure.Storage.Blobs_.
+2. Add the Orleans provider packages for those Aspire components. In our example, we are using _Microsoft.Orleans.Persistence.AzureStorage_ and _Microsoft.Orleans.Clustering.AzureStorage_
+3. Add Orleans to the host application builder.
+
+In OrleansServer's _Program.cs_ file, we must configure the Aspire components we are using and add Orleans to the host builder. Note that the names provided must match the names used in the Aspire AppHost project: "clustering" for the clustering provider, and "grainstate" for the grain state storage provider:
+
+```csharp
+using Orleans.Runtime;
+using OrleansContracts;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+builder.AddKeyedAzureTableClient("clustering");
+builder.AddKeyedAzureBlobClient("grainstate");
+builder.UseOrleans();
+
+var app = builder.Build();
+
+app.MapGet("/", () => "OK");
+
+await app.RunAsync();
+
+public sealed class CounterGrain(
+    [PersistentState("count")] IPersistentState<int> count) : ICounterGrain
+{
+    public ValueTask<int> Get()
+    {
+        return ValueTask.FromResult(count.State);
+    }
+
+    public async ValueTask<int> Increment()
+    {
+        var result = ++count.State;
+        await count.WriteStateAsync();
+        return result;
+    }
+}
+```
+
+We do similarly in the OrleansClient project, adding the Aspire resources which our project needs to join the Orleans cluster as a client, and configuring the host builder to add an Orleans client:
```

Comment:
There is no "we" or "our" in docs, it's all about the reader, "you" and "your".

```suggestion
You do similarly in the OrleansClient project, adding the .NET Aspire resources which your project needs to join the Orleans cluster as a client, and configuring the host builder to add an Orleans client:
```

Reply:
That's not very inclusive now, is it